src/Route/RoutePageGenerator.php

```diff
@@ -249,4 +239,39 @@ protected function writeln(OutputInterface $output = null, $message)
             $output->writeln($message);
         }
     }
+
+    /**
+     * Generate root page (parent for another generated pages).
+     * If root path available in config - create from config
+     * Else - generate it statically.
+     *
+     * @return object root page
```

Comment:
Sure, this is just an `object`?

Reply:
Not sure, can you tell me, what is better to write?

Reply:
PageInterface